core/src/main/java/org/elasticsearch/script/Script.java

```diff
@@ -149,7 +132,11 @@ public String getLang() {
     }
 
     @Override
-    public final void readFrom(StreamInput in) throws IOException {
+    public final Script readFrom(StreamInput in) throws IOException {
+        return new Script(in);
+    }
+
+    public Script(StreamInput in) throws IOException {
```

Comment:
I've been moving these up, right under the other constructors and moving writeTo under it.